cmd/litcli/autopilot.go

```diff
@@ -129,6 +129,27 @@ var addAutopilotSessionCmd = cli.Command{
 				`An empty rule map is allowed with {} to ` +
 				`use the default rules.`,
 		},
+		cli.Int64Flag{
```

Comment:
wondering how useful this actually is cause the user would need to manually go workout the resulting int. 

How about instead having a flag per `Clear**` `PrivacyFlag` type that we have at the moment and then construct the int on behalf of the user?

Reply:
so that I can do like `--clear-pubkeys --clear-htlcs` instead of needing to figure out what each int & then doing a manual bitwise OR

Reply:
right, I had the same thought, but decided against it because it adds a lot of noise to the command documentation and because of reasoning the same way we did previously, thinking it would be mostly used for testing. Another way could be to parse it from the string representation like `--privacy-flags="ClearPubkey|ClearAmounts"`. I would add one of the solutions if you find it important

Reply:
cool yeah the "flags="ClearPubkey|ClearAmounts" seems nice. Even if this is just for testing, it is nice if it is easy to quickly get something going (bitwise ORing manually is a bit much)

Reply:
what's nice about your idea to do the "flags="ClearPubkey|ClearAmounts"" way is also that we dont have to keep updating the CLI code each time we add a new one

Reply:
I implemented and tested string-based flags :+1: